syntax = "proto3";
option go_package="/search_vector;";

message SearchVectorRequest{
    // @inject_tag: form:"query" uri:"query"
    repeated string query = 1;
}

message SearchVectorResponse{
    int64 code = 1;
    repeated string doc_ids = 2;
    string msg = 3;
    string error = 4;
}

service SearchVectorService {
    rpc SearchVector(SearchVectorRequest) returns(SearchVectorResponse);
}